### PR TITLE
fix case bug and string number bug in qa

### PR DESCRIPTION
**Pull request by ywrsusan**

### CODE REVIEW

File: static/src/components/fieldsconfig/consensus_config.vue [hunk 1]
```diff
@@ -105,17 +105,17 @@ export default {
...
     _isIntegerNumeric: function (n) {
-        var _n = Number(n);
+        var _n = parseInt(n, 10);
         return Math.floor(_n) === _n;
     },
 
     _write: function () {
-        if (!this._isIntegerNumeric(this.threshold) || this.threshold <= 50 ||
-                this.threshold > 100) {
+        if (!this._isIntegerNumeric(this.consensusThreshold) || this.consensusThreshold <= 50 ||
```
Comment (jayflo) [inline static/src/components/fieldsconfig/consensus_config.vue]: Just double checking: do you need to cast e.g. `this.consensusThreshold` to int here?  Like you do here: https://github.com/bloomberg/pybossa-default-theme/pull/118/files#diff-1e8c3142d2cb9be9f01400dcc7555c71R134

**Comment by jayflo** [comment]
Otherwise looks good for me.

File: static/src/components/fieldsconfig/consensus_config.vue (hunk 1 above)
Comment (ywrsusan) [reply static/src/components/fieldsconfig/consensus_config.vue]: I didn't cast it to int here, just check whether the input contains letters or is float number.

File: static/src/components/fieldsconfig/field_config_base.vue [hunk 2]
```diff
@@ -108,12 +108,12 @@ export default {
 
     changeRetryStatus () {
       this.retry = !this.retry;
-      this.$emit('retryForConsensus', this.retry);
       this.changeRetryConfig({ name: this.name,
                               retry: this.retry });
     },
 
     changeCollapseStatus () {
+      console.log(this.retry);
```
Comment (mariarivera30) [inline static/src/components/fieldsconfig/field_config_base.vue]: I think this line can be removed

File: static/src/components/fieldsconfig/store.js [hunk 3]
```diff
@@ -44,12 +51,12 @@ const storeSpecs = {
     },
 
     hasConsensusConfig (state) {
-      return (state.consensusConfig && state.consensusConfig.consensusThreshold);
+      return (state.consensusConfig && state.consensusConfig.consensusThreshold > 0);
```
Comment (mariarivera30) [inline static/src/components/fieldsconfig/store.js]: Why this line changed to >0

File: static/src/components/fieldsconfig/store.js [hunk 4]
```diff
@@ -81,14 +87,17 @@ const storeSpecs = {
     },
 
     changeRetryConfig (state, { name, retry }) {
-      state.answerFields[name].retryForConsensus = retry;
+      state.answerFields[name]['retry_for_consensus'] = retry;
     },
 
     setData (state, { csrf, answerFields, consensus }) {
+      console.log(answerFields);
```
Comment (mariarivera30) [inline static/src/components/fieldsconfig/store.js]: This line can be removed

**Pull request merged**